scripts/cypress.js

```diff
@@ -15,7 +15,10 @@ const baseConfig = {
   testFiles: ['**/e2e/**/*.e2e.ts'],
   video: false,
   screenshotOnRunFailure: false,
-  baseUrl: 'http://localhost:3000',
+  baseUrl: process.env.DEPLOYBASEPATH
+    ? // Base path hard coded for converged for now, can be modified to be configurable if required to other projects
+      `https://${process.env.DEPLOYHOST}/${process.env.DEPLOYBASEPATH}/react-components/storybook`
```

Comment:
```suggestion
  baseUrl: process.env.DEPLOYURL
    ? // Base path hard coded for converged for now, can be modified to be configurable if required to other projects
      `${process.env.DEPLOYURL}/react-components/storybook`
```

nit: let's use `DEPLOYURL` instead

https://github.com/microsoft/fluentui/blob/01568189099e96de8f2e789be3316803162e307e/.devops/templates/variables.yml#L19-L20